src/wrapped/wrappedlibsqlite3.c

```diff
@@ -25,8 +25,40 @@ const char* libsqlite3Name = "libsqlite3.so.0";
 
 #include "generated/wrappedlibsqlite3types.h"
 
-//#include "wrappercallback.h"
+#include "wrappercallback.h"
 
-// Insert my_* functions here...
+#define SUPER() \
+GO(0)   \
+GO(1)   \
+GO(2)   \
+GO(3)   \
+GO(4)
+
+// sqlite3_exec ...
+#define GO(A) \
+static uintptr_t my_sqlite3_exec_fct_##A = 0; \
+static int* my_sqlite3_exec_##A(void* a, int b, char** c, char** d) { \
```

Comment:
Type of the function is
```C
int sqlite3_exec(
  sqlite3*,                                  /* An open database */
  const char *sql,                           /* SQL to be evaluated */
  int (*callback)(void*,int,char**,char**),  /* Callback function */
  void *,                                    /* 1st argument to callback */
  char **errmsg                              /* Error msg written here */
);
```
return type if the callback is `int` not `int*`

Reply:
Sorry for my mistake. I have fixed it in my newest commit.